toolkit/resources/assets/grub2/grub

```diff
@@ -0,0 +1,23 @@
+GRUB_TIMEOUT=0
+GRUB_DISTRIBUTOR="AzureLinux"
+GRUB_DISABLE_SUBMENU=y
+GRUB_TERMINAL_OUTPUT="console"
+GRUB_CMDLINE_LINUX="{{.LuksUUID}} {{.LVM}} {{.IMAPolicy}} {{.ReadOnlyVerityRoot}} {{.SELinux}} rd.auto=1 lockdown=integrity sysctl.kernel.unprivileged_bpf_disabled=1 loglevel=3 {{.CGroup}}"
+GRUB_CMDLINE_LINUX_DEFAULT="{{.ExtraCommandLine}} $kernelopts"
+    
+# =============================notice===============================
+# IMPORTANT: package and feature-specific behaviors are defined in
+#   /etc/default/grub.d/*.cfg. The cfg files are sourced last
+#   before grub2-mkconfig is called and hence have higher precedence
+#   than this file's GRUB_CMDLINE_LINUX. The order of sourcing is:
+#     - GRUB_CMDLINE_LINUX
+#     - /etc/default/grub.d/*.cfg
+#     - GRUB_CMDLINE_LINUX_DEFAULT 
+#   ...so GRUB_CMDLINE_LINUX_DEFAULT is applied last and thus is
+#   the most final, followed by the settings in grub.d
```

Comment:
This last bit doesn't flow quite right to me. I get the point of it, but it took me a few reads.